dataikuapi/dss/project.py

```diff
@@ -856,9 +855,10 @@ def get_jupyter_notebook(self, notebook_name):
         :returns: A handle to interact with this jupyter notebook
         :rtype: :class:`~dataikuapi.dss.notebook.DSSNotebook` jupyter notebook handle
         """
-        notebook_content = self.client._perform_json("GET",
-                                             "/projects/%s/jupyter-notebooks/%s" % (self.project_key, notebook_name))
-        return DSSNotebook(self.client, self.project_key, notebook_name, content=notebook_content)
+        notebook = DSSJupyterNotebook(self.client, self.project_key, notebook_name)
+        # Force content download
+        notebook.get_content()
```

Comment:
Why forcing to download content? Lazy loading is nice ;-)

Reply:
it was to keep consistency with the previous usage.
I will remove it, then.